docs/content/preview/develop/build-apps/python/cloud-ysql-python.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Build a Python application that uses YSQL
+headerTitle: Build a Python application
+linkTitle: Python
+description: Build a simple Python application using the Python psycopg2 driver and using the YSQL API to connect to and interact with a YugabyteDB Managed cluster.
+headContent: "Client driver: psycopg2"
+aliases:
+  - /preview/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-python/
+menu:
+  preview:
+    parent: build-apps
+    name: Python
+    identifier: cloud-python
+    weight: 300
+type: docs
+---
+
+The following tutorial shows a small [Python application](https://github.com/yugabyte/yugabyte-simple-python-app) that connects to a YugabyteDB cluster using the [Python psycopg2 PostgreSQL database adapter](../../../../reference/drivers/ysql-client-drivers/#psycopg2) and performs basic SQL operations. Use the application as a template to get started with YugabyteDB Managed in Python.
```

Comment:
Maybe link to preview/reference/drivers/python/postgres-psycopg2-reference/ page instead; same with Learn more